subprojects/docs/docs.gradle

```diff
@@ -287,8 +287,6 @@ samples {
                 from(templates.javaModuleInfoForListLibrary)
                 from(templates.javaModuleInfoForUtilitiesLibrary)
                 from(templates.javaModuleInfoForApplication)
-                from(templates.javaJunit5TestForApplication)
-                from(templates.javaJunit5TestForListLibrary)
```

Comment:
:x: These should not be removed

Reply:
Sorry, I meant to double check the merge conflict and forgot :| Thanks for catching